pkg/restapi/vc/operation/persistence.go

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
I'm guessing this file will be used once the EDV integration is done?

Reply:
for creating verifying credential you need cred ID this file help us to create the issuer profile which will be the first step before creating credential. Does that answer your question ?